swap_meet/item.py

```diff
@@ -1,2 +1,29 @@
+## I attempted to define these variables below in Item instead of globally, 
+## but it made the code less readable, and required calling them with Items.* or self.*
+
+CATEGORIES = {
+    "": "Hello World!",
+    "Clothing": "The finest clothing you could wear.",
+    "Decor": "Something to decorate your space.",
+    "Electronics": "A gadget full of buttons and secrets.",
+}
+
+CONDITION_DESCRIPTORS = {
+    0: "Oof. Are you sure you want this?",
+    1: "Wow. It's pretty bad.",
+    2: "Significant damage.",
+    3: "Needs some love.",
+    4: "Minor signs of wear or damage.",
+    5: "So good it might be a scam.",
+}
```

Comment:
Heather great job making these global "constant" variables. This helps eliminate redundant code blocks and makes this accessible to other subclasses.